src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -123,6 +122,7 @@ void CascadiaSettings::_CreateDefaultProfiles()
     Profile defaultProfile{};
     defaultProfile.SetFontFace(L"Consolas");
     defaultProfile.SetCommandline(L"cmd.exe");
+    defaultProfile.SetStartingDirectory(L"%HOMEDRIVE%%HOMEPATH%");

```

Comment:
HOMEDRIVE and HOMEPATH may not be set in all scenarios.  If you look at powershell core it has to deal with this today.  So unless an empty result here is valid, this logic probably needs to be centralized.

Reply:
See notes here:
https://stackoverflow.com/posts/36392591/revisions